docker-compose.yml

```diff
@@ -63,11 +63,12 @@ services:
   speedtest:
     tty: true
     stdin_open: true
+    privileged: true
```

Comment:
I've been running this under the `pi` user on a number of Pis without any issue. I typically avoid giving `privileged` unless absolutely necessary (and ideally for software I have a lot more control over).

Can you give more details (maybe in an issue) about the problem you're hitting? What user account you're using on the Pi? How you installed Docker, etc.?

Reply:
I created #17 with more info. 
I will do a test starting from scratch to eliminate OS issues. Should have done that before the PR.